Makefile

```diff
@@ -123,6 +141,10 @@ uninstall-conf:
 	rm -f $(prefix)$(sysconfdir)/mender/artifact_info
 	-rmdir -p $(prefix)$(sysconfdir)/mender
 
+uninstall-dbus:
+	rm -f $(prefix)$(datadir)/dbus-1/system.d/io.mender.conf
```

Comment:
Are you sure about the file name convention (from what I see, it should be the name of the object we register, not its prefix) and the path? On a Debian stretch machine, the conf files are here: /etc/dbus-1/system.d/

Reply:
> Hmm, the failure reveals a bit bigger problem: This needs to be runnable locally without running as root. I'll see if it's possible to adapt the tests to use the session bus during testing.

Unfortunately, I failed to use the session dbus because it requires X11 running. I could have done something wrong, I don't know. But the error I see in CI/CD is different:

```
$ cp dbus-1/system.d/mender.conf /etc/dbus-1/system.d/mender.conf
cp: cannot stat 'dbus-1/system.d/mender.conf': No such file or directory
```

Reply:
Yes, this error is easy to fix. But it's the fact that it copies into a root-owned directory which is the problem. I want to understand this a bit better, so I will spend a bit more time looking for ways to avoid that. If nothing else, I will learn about DBus. :-)

Reply:
> Are you sure about the file name convention (from what I see, it should be the name of the object we register, not its prefix) and the path?

I think you're right, it should be `io.mender.AuthenticationManager.conf`, right?

> On a Debian stretch machine, the conf files are here: /etc/dbus-1/system.d/

Hmm, I don't know. There is a mix on my machine. Intuitively, it seems more correct to put them in `/usr/share`, so that you can override them from `/etc`. I think what is written [here](https://manpages.debian.org/experimental/dbus/dbus-daemon.1.en.html#CONFIGURATION_FILE) supports this view. What do you think?

Reply:
Man page of dbus-deamon says:
```
       This is intended to allow extension of the system bus by particular packages. For example,
       if CUPS wants to be able to send out notification of printer queue changes, it could
       install a file to /usr/share/dbus-1/system.d or /etc/dbus-1/system.d that allowed all apps
       to receive this message and allowed the printer daemon user to send it.
```

I believe your path is good, let's stick to it.